src/lib/ngsi10/QueryContextRequestVector.cpp

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
This is actualy a ngsi10 class? Or a "helper" class maybe better to place in a different directory?